### PR TITLE
Feature/gh 987 centralized colors

### PR DESCRIPTION
**Pull request by DmitryBespalov**
- Centralized color palette in Colors.txt (one file)
- Replaced old colors with new colors according to rebranding map
- Replaced UIColor.[white,black,darkText,clear] with colors from the palette

### CODE REVIEW

File: SafeUIKit/SafeAppUI/ReusableDetachedControllers/PaymentMethod/BasePaymentMethodViewController.swift [hunk 1]
```diff
@@ -20,16 +20,16 @@ class BasePaymentMethodViewController: UIViewController, UITableViewDelegate, UI
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        view.backgroundColor = .white
```
Comment (elgatovital) [inline SafeUIKit/SafeAppUI/ReusableDetachedControllers/PaymentMethod/BasePaymentMethodViewController.swift:23]: Not critical but I like this short notation better: .white vs ColorName.white.color

File: SafeUIKit/SafeUIKit/Components/BackgroundImageView/BackgroundImageView.swift [hunk 2]
```diff
@@ -24,18 +24,20 @@ public final class BackgroundImageView: UIImageView {
...
         dimmedView.frame = frame
         insertSubview(dimmedView, at: 0)
     }
 
     public var isDimmed = false {
         didSet {
-            dimmedView.backgroundColor = isDimmed ? UIColor.black.withAlphaComponent(0.15) : .clear
+            dimmedView.backgroundColor = isDimmed ?
+                ColorName.black.color.withAlphaComponent(0.15) :
+                ColorName.transparent.color
```
Comment (elgatovital) [inline SafeUIKit/SafeUIKit/Components/BackgroundImageView/BackgroundImageView.swift]: Maybe use some global constant for dimming value 0.15?

**Review by elgatovital — APPROVED** [review]

File: SafeUIKit/SafeAppUI/ReusableDetachedControllers/PaymentMethod/BasePaymentMethodViewController.swift (hunk 1 above)
Comment (DmitryBespalov) [reply SafeUIKit/SafeAppUI/ReusableDetachedControllers/PaymentMethod/BasePaymentMethodViewController.swift:23]: this changed on purpose. The short `.white` is a `UIColor.white` constant, but I need it to be a `ColorName.<color>.color` constant.

File: SafeUIKit/SafeUIKit/Components/BackgroundImageView/BackgroundImageView.swift (hunk 2 above)
Comment (DmitryBespalov) [reply SafeUIKit/SafeUIKit/Components/BackgroundImageView/BackgroundImageView.swift]: yes, will add the black15 color

**Pull request merged**